package.json

```diff
@@ -72,7 +72,7 @@
     "rimraf": "^2.5.4",
     "tslint": "latest",
     "tslint-test-config-non-relative": "file:test/external/tslint-test-config-non-relative",
-    "typescript": "^2.3.0"
+    "typescript": "next"
```

Comment:
@nchen63 has another approach to fixing master that compiles with ts 2.3: https://github.com/palantir/tslint/tree/fix-master